### PR TITLE
oslogin test: update tests based on new authorization implementation

### PR DESCRIPTION
**Pull request by dorileo**
The new implementation removes the use of one of the PAM modules and introduces AuthorizedPrincipalsCommand.

### CODE REVIEW

File: imagetest/test_suites/oslogin/oslogin_test.go [hunk 1]
```diff
@@ -99,7 +111,7 @@ func TestOsLoginDisabled(t *testing.T) {
 		t.Fatalf("cannot read /etc/pam.d/sshd")
 	}
 	contents := string(data)
-	if strings.Contains(contents, "pam_oslogin_login.so") || strings.Contains(contents, "pam_oslogin_admin.so") {
+	if strings.Contains(contents, "pam_oslogin_login.so") {
```
Comment (ericdand) [inline imagetest/test_suites/oslogin/oslogin_test.go:114]: No more "pam_oslogin_admin.so"?

Comment (dorileo) [reply imagetest/test_suites/oslogin/oslogin_test.go:114]: Nope, it was removed, remember?

Comment (dorileo) [reply imagetest/test_suites/oslogin/oslogin_test.go:114]: https://github.com/GoogleCloudPlatform/guest-oslogin/pull/116/commits/39240b36b11a146229355f1fcaa22150d11b011c

Comment (ericdand) [reply imagetest/test_suites/oslogin/oslogin_test.go:114]: Ah, right you are. Sounds good.

**Pull request merged**